manuskript/models/abstractItem.py

```diff
@@ -141,6 +141,9 @@ def removeChild(self, row):
         @return: the removed abstractItem
         """
         r = self.childItems.pop(row)
+        # Disassociate the child from its parent and the model.
```

Comment:
Dissociate instead of disassociate ?

Reply:
They are both valid for as far I am able to tell based on [this link](https://grammarist.com/usage/dissociate-disassociate/). To be honest, I didn't even know 'dissociate' was valid english myself until you asked this.

Can change or can keep, I don't really care. For me they are both english, although the form I used obviously makes more sense to me.

Reply:
Interesting read! Never heard disassociate, sounded weird to me. If it's not a typo, then all good! 👍